android/assets/jsons/translations/Traditional_Chinese.properties

```
@@ -4927,11 +4927,11 @@ Modern Armor = 現代坦克
 Jet Fighter = 噴氣式戰鬥機
 
  # Requires translation!
-Giant Death Robot = 
+Giant Death Robot = 巨型死亡機器人
```

Already forgotton about how Traditional Chinese translator in civ5 translated it, but I know the Simplified_Chinese had a much better translation.
```suggestion
Giant Death Robot = 末日機甲
```

---

I agree.